source/_integrations/netgear.markdown

```diff
@@ -17,15 +18,48 @@ ha_codeowners:
 ha_ssdp: true
 ---
 
-This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device.
+This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device and control the NETGEAR device.
+Both routers and access points can be used with this integration. Some access points will not be automatically discovered and need to be set up manually.
 
 {% include integrations/config_flow.md %}
 
-Most NETGEAR routers use port 5000 to communicate, however the following list of models are known to use port 80:
-- Nighthawk RAX50
-- Nighthawk X4S - AC2600 (R7800)
-- Orbi
-- XR500
-When setup through ssdp discovery the port schould be automatically detected.
+## Optional settings
```

Comment:
We have `{% include integrations/option_flow.md %}` for that nowadays, please check e.g., the `cast` integration documentation page on how to use it.

Reply:
brilliant, that is perfect, thank you!